piton/tools/src/proto/block.list

```diff
@@ -25,7 +25,7 @@
 # Format:
 # BlockID                   BlockPath                           Supported Board,Frequency(MHz),DDRSize(Mbytes)
 piton_aws                   ../../build/f1/piton_aws/design                         f1,62.5,4096
-system                      .                                                       vc707,60,1024;genesys2,66.667,1024;nexysVideo,30,512;vcu118,100,2048;xupp3r,60,32768
+system                      .                                                       vc707,60,1024;genesys2,25,1024;nexysVideo,30,512;vcu118,100,2048;xupp3r,60,32768
```

Comment:
Still not excited about this - do we have a sense of the manually retimed FPU's timing?